### ISSUE TITLE
Accessibility issues: Form elements must have aria-label and aria-roles must contain particular children

### ISSUE THREAD

**Issue by ligiacsadi** [moroshko/react-autosuggest#476]
Hi, 
I am using the Google Chrome extension aXe to analyse and find any accessibility defects in the web application that I am developing. Unfortunately, the application finds a few accessibility violations linked to the autosuggest component. I have taken a screenshot of the two issues that I am currently stuck on and which are quite critical. 

**1. Form elements must have aria-label:** 
![image](https://user-images.githubusercontent.com/33862543/33066443-310116ac-cea3-11e7-8139-f2eb951db1d8.png)

**2. ARIA-roles must contain particular children:** 
![image](https://user-images.githubusercontent.com/33862543/33066524-6a9052fc-cea3-11e7-8729-1cfce85b4aff.png)

**react-autosuggest/dist/standalone/autosuggest.js**. I tried to locate it at this location but was unable to. 
![image](https://user-images.githubusercontent.com/33862543/33066903-74d0b922-cea4-11e7-890b-3406e6daca90.png)

Seems to me that the autosuggest component is missing **aria-label** and **listbox child role** from the source code. Since I cannot pass these in as a parameters or add to my divs, would you be so kind to add them to the autosuggest code ? I would really, really appreciate it. 

Thank you very much.
Ligia

**Comment by moroshko** [comment]
For the first issue, try to add a `<label>` outside of the Autosuggest, and set its `htmlFor` to be equal to the Autosuggest's `inputProps.id`.

For the second issue, could you provide the "Learn more" link to see details. I'm not sure at this point what the requirement is.

**Comment by ligiacsadi** [comment]
Thank you so much for your response @moroshko . Your suggestion worked and it fixed the second issue I was seeing as well.
Thanks again.

**Comment by judygab** [comment]
Adding label didn't work for me. The error remains the same that autosuggest container does not have an aria-label. Is there any other way to approach this?
